SYCL/KernelFusion/non_unit_local_size.cpp

```diff
@@ -0,0 +1,82 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// UNSUPPORTED: cuda || hip
+// REQUIRES: fusion
+
+// Test complete fusion with local internalization specified on the
+// accessors, where each work-item processes multiple data-items.
+
```

Comment:
Is implementation doing anything specific to support this? Or is it here mostly as an example/documentation?

Reply:
This test is mainly making sure that the size of the local memory is correctly calculated if the ratio between the number of workitems and the number of elements in the internalized buffer is not 1:1.